GEOSldas_App/util/postproc/ObsFcstAna_stats/user_config.py

```diff
@@ -0,0 +1,132 @@
+
+import sys;       sys.path.append('../../shared/python/')
+import warnings;  warnings.filterwarnings("ignore")
+import os
+
+import numpy as np
+
+from datetime               import datetime, timedelta
+from read_GEOSldas          import read_tilecoord, read_obs_param
+from postproc_ObsFcstAna    import postproc_ObsFcstAna
+
+def get_config():
+
+    # =========================================================================================
+    #
+    # User-defined inputs for post-processing of ObsFcstAna output
+
+    # Range of months to process:
+    
+    start_year  = 2015
+    start_month =    4
+    last_year   = 2016
+    last_month  =    4
+
+    # Sums or stats will be processed for exp_main:
+
+    exp_main = {
+        'expdir'        : '/discover/nobackup/projects/gmao/merra/iau/merra_land/SMAP_runs/SMAP_Nature_v11/',
+        'expid'         : 'DAv8_SMOSSMAP',
+        'exptag'        : 'DAMulti_SMAP', 
+        'domain'        : 'SMAP_EASEv2_M36_GLOBAL',
+        'da_t0'         : 3,                              # hour of first land analysis
+        'da_dt'         : 10800,                          # ObsFcstAna file interval in seconds
+        'species_list'  : [5,6,7,8],
```

Comment:
I'm not sure I fully understand "species_list".  Is this an index vector into the species defined in "obsparam"?  That is, from exp_main, we want the stats for the 5th, 6th, 7th, and 8th species within the "obsparam" vector?  I forget if we enforce that the species ID in the obsparam vector is always a consecutive number from 1 to N_obsparam.  If that's the case, the indices here match the species "ID" in the obsparam.  But I'm not sure this is generally the case.  So further tweaking of the meaning of "species_list" is probably required in the py script.

Reply:
This is indeed the case as far as I'm aware. For example, we have obsparam IDs 1-4 for SMOS species and 5-8 for SMAP species in DA_SMOSSMAP, and 1-4 for SMAP species in DA_SMAP. The order is based on the ensupd input. Maybe @amfox37 can confirm.